dashboard/templates/base.html

```diff
@@ -32,7 +32,6 @@
     </header>
     {% block content %}
     {% endblock %}
-    <footer><hr>Copyright 2018 University of Michigan ITS Teaching and Learning</footer>
```

Comment:
Did you mean to remove the footer entirely in this PR?
I think it should match up with other sites and be

> © 2018 The Regents of the University of Michigan


Rather than what we have here. But we should have something.

Reply:
updated to "Copyright © 2018 The Regents of the University of Michigan"